regional_mom6/regional_mom6.py

```diff
@@ -619,13 +886,187 @@ def _make_vgrid(self):
+            # datasets = [xr.open_dataset(file) for file in all_files]
+            return all_files
+        except:
+            return "Error retrieving ocean state files"
+
+    @property
+    def tides_boundaries(self):
+        """
+        Read the tides from disk, and print 'em
+        """
+        tides_path = self.mom_input_dir / "forcing"
+        try:
+            # Use glob to find all tides files
+            patterns = ["regrid*", "tu_*", "tz_*"]
+            all_files = []
+            for pattern in patterns:
+                all_files.extend(glob.glob(os.path.join(tides_path, pattern)))
+                all_files.extend(glob.glob(os.path.join(self.mom_input_dir, pattern)))
+
+            if len(all_files) == 0:
+                return "No tides files set up yet (or files misplaced from {}). Call `setup_tides_boundaries` method to set up tides.".format(
+                    tides_path
+                )
+
+            # Open the files as xarray datasets
+            # datasets = [xr.open_dataset(file) for file in all_files]
+            return all_files
+        except:
+            return "Error retrieving tides files"
+
+    @property
+    def era5(self):
+        """
+        Read the era5's from disk, and print 'em
+        """
+        era5_path = self.mom_input_dir / "forcing"
+        try:
+            # Use glob to find all *_ERA5.nc files
+            all_files = glob.glob(os.path.join(era5_path, "*_ERA5.nc"))
+            if len(all_files) == 0:
+                return "No era5 files set up yet (or files misplaced from {}). Call `setup_era5` method to set up era5.".format(
+                    era5_path
+                )
+
+            # Open the files as xarray datasets
+            # datasets = [xr.open_dataset(file) for file in all_files]
+            return all_files
+        except:
+            return "Error retrieving ERA5 files"
+
+    @property
+    def initial_condition(self):
+        """
+        Read the ic's from disk, and print 'em
+        """
+        forcing_path = self.mom_input_dir / "forcing"
+        try:
+            all_files = glob.glob(os.path.join(forcing_path, "init_*.nc"))
+            all_files = glob.glob(os.path.join(self.mom_input_dir, "init_*.nc"))
+            if len(all_files) == 0:
+                return "No initial conditions files set up yet (or files misplaced from {}). Call `setup_initial_condition` method to set up initial conditions.".format(
+                    forcing_path
+                )
+
+            # Open the files as xarray datasets
+            # datasets = [xr.open_dataset(file) for file in all_files]
+            # return datasets
+
+            return all_files
+        except:
+            return "No initial condition set up yet (or files misplaced from {}). Call `setup_initial_condition` method to set up initial conditions.".format(
+                self.mom_input_dir / "forcing"
+            )
+
+    @property
+    def bathymetry_property(self):
```

Comment:
Just wondering if we need the _property suffix? Or would calling `expt.bathymetry` be a bit more intuitive to match the others like `expt.initial_condition`?